list.go

```diff
@@ -116,8 +117,22 @@ func getContainers(context *cli.Context) ([]containerState, error) {
 				ID:             state.BaseState.ID,
 				InitProcessPid: state.BaseState.InitProcessPid,
 				Status:         containerStatus.String(),
+				Bundle:         getBundlePath(state.Config.Labels),
 				Created:        state.BaseState.Created})
 		}
 	}
 	return s, nil
 }
+
+func getBundlePath(labels []string) string {
```

Comment:
Since this is for metadata in general ... should probably pass "bundle" to this function, name the function getMetaData and return an error if it's not found..